app/views/admin/reports/index.html.haml

```diff
@@ -50,15 +52,22 @@
       - reports.each do |report|
         .report-card__summary__item
           .report-card__summary__item__reported-by
-            - if report.account.instance_actor?
-              = site_hostname
-            - elsif report.account.local?
-              = admin_account_link_to report.account
-            - else
-              = report.account.domain
+            = link_to admin_report_path(report), class: 'report-card__summary__item__report-id' do
+              .one-line= t('admin.reports.report', id: report.id)
```

Comment:
This whole case block ... 1) maybe dump into helper method, 2) are there existing i18n strings to use?